src/selectPackage/selectPackage.html

```diff
@@ -30,9 +30,9 @@
               If successful
             </div>
           </div>
-          <div class="button button1">
-            <a href="/newSeed">START REGISTRATION</a>
-          </div>
+          <button click.delegate="navigate('newSeed')" class="button button1">
```

Comment:
![image](https://user-images.githubusercontent.com/1821666/133450917-c3740748-9312-4f90-87a9-baadc74b39e8.png)

Reply:
I think the best way to fix this is to leave it as an anchor button but apply the .button1 style to it.

note that a ".button" style doesn't even exist, which is by design.